woocommerce/woocommerce-android
# EU-bound Shipping Notice: Introduce EU customs rule check

Summary
==========
Fix issue #8889 and #8890 by introducing the expected scenario of when the EU notice should be presented. Following the USPS instructions, the notice will be triggered when the origin address is from the US and the destination address is one of the following countries:

<img width="697" alt="image" src="https://user-images.githubusercontent.com/5920403/235547782-54e2b1de-879a-4c8e-9568-1e9e2fa8d03d.png">

Screen Capture
==========
https://user-images.githubusercontent.com/5920403/235547826-5cd451ee-2236-49b1-90e4-72b01bdc4734.mp4

How to Test
==========
For better testing the scenarios introduced here, I recommend setting the `AppPrefs.isEUShippingNoticeDismissed` always to return `false`.

1. Open the app using a store with the `Shipping Labels` plugin activated
2. Open the Order Details of an order with the `Processing` status
3. Hit the `Create Shipping Label` button
4. Verify that the Banner only appears when the Origin country is `US` and the Destination country is one of the EU countries listed above.
5. Verify that the `Customs` view also follows the exact same visibility configuration from the `Create Shipping Label` view

Update release notes:
